baselines/dom.generated.d.ts

```diff
@@ -16233,6 +16233,7 @@ interface HTMLElementTagNameMap {
 }
 
 interface SVGElementTagNameMap {
+    "a": SVGAElement;
```

Comment:
this is gonna cause a build error, cause now the two map types are not matching.. i believe this is what is failing the build now. i remember we tried that before and chose not to do it.

Reply:
This is because of old deprecated ElementTagNameMap extends both HTML/SVGElementTagNameMap...